src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,103 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int CAPACITY_COEFFICIENT = 2;
```

Comment:
```suggestion
    private static final int GROW_FACTOR = 2;
```
IMO not clear naming